qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/create/CreateCustomerInput.groovy

```diff
@@ -1,13 +1,12 @@
 package life.qbic.portal.portlet.customers.create
 /**
- * <short description>
+ * Input interface for the {@link life.qbic.portal.portlet.customers.create.CreateCustomer} use case
  *
- * <detailed description>
+ * This interface describes the methods the use case exposes to its caller.
```

Comment:
```suggestion
 * This interface describes the sequential steps how to create a new customer at QBiC.
```

Reply:
It does not really provide any sequence.

Reply:
true